controller/volume_controller.go

```diff
@@ -1787,15 +1787,24 @@ func (c *VolumeController) openVolumeDependentResources(v *longhorn.Volume, e *l
 
 			// If the engine isn't attached or the node goes down, the replica can be marked as failed.
 			// In the attached mode, we can determine whether the replica can fail by relying on the data plane's connectivity status.
-			nodeDeleted, err := c.ds.IsNodeDeleted(r.Spec.NodeID)
+			//
+			// In a scenario when the Kubelet is inactive for an extended period, the taint manager could potentially
```

Comment:
Do we know what is `extended period`? how long is it?

Reply:
When the Kubelet is stopped, the node will be tainted NoExecute and the taint manager deletes all pods that are not tolerating the NoExecute taint or passed the TolerationSeconds. Hence this means it could be immediately after Kubelet is started.